src/dune_rules/dune_project.mli

```diff
@@ -158,6 +158,7 @@ val accept_alternative_dune_file_name : t -> bool
 val strict_package_deps : t -> bool
 val cram : t -> bool
 val info : t -> Package.Info.t
+val warnings : t -> Warning.Settings.t
```

Comment:
NP: Can we please call this `warning_settings` instead

Reply:
But the field in the project file is called `warnings`, so that would be a little more confusing imo. No strong preference though.

Reply:
Ah ok that makes sense.